src/main/scala/nl/knaw/dans/easy/multideposit/package.scala

```diff
@@ -124,7 +124,6 @@ package object multideposit {
       * Writes a CharSequence to a file creating the file if it does not exist using the default encoding for the VM.
       *
       * @param data the content to write to the file
-      * @throws IOException in case of an I/O error
```

Comment:
I think the addition of the annotations makes it kind of equivalent to the original. So definitely 👍 for that!